src/cards/ha-weather-card.js

```diff
@@ -271,6 +271,10 @@ class HaWeatherCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
     return localize(`state.weather.${state}`) || state;
   }
 
+  computeName(stateObj) {
+    return this.config.name || stateObj.attributes.friendly_name;
```

Comment:
We shouldn't use `stateObj.attributes.friendly_name` but instead use https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/common/entity/compute_state_name.ts

Reply:
`this.config` is not set on legacy UI, so can't just be referenced for any element in `src/cards`